build_docs.py

```diff
@@ -388,7 +395,7 @@ def setup_switchers(
     - Cross-link various languages in a language switcher
     - Cross-link various versions in a version switcher
     """
-    language_pairs = sorted((l.tag, l.name) for l in languages if l.in_prod)
+    language_pairs = sorted((l.tag, l.switcher_label) for l in languages if l.in_prod)
```

Comment:
I suggest sorting by name/switcher label instead of prioritizing the tag. This way it will be easier to find by English name lexicographically when the list will grow longer.